### PR TITLE
fix(cli): correct template-not-found hint to `fenn list`

### LINKED ISSUE #312: Bug: Incorrect Error Response for Template Not found
### Describe the bug

Whenever a user enters the incorrect name for a template, the CLI prints out the error and asks the user to use the command "fenn pull --list" which isn't correct and doesn't work

### How to reproduce

Type "fenn pull DOES_NOT_EXIST temp" to the CLI

### Expected behavior

Instead the response should be "fenn list" to get all the templates

### Observed behavior

Previously mentioned

### Screenshots

_No response_

### Environment

- OS: MacOS
- Python Version 3.13.5

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rusetiq**
## Summary

Fixes #312.

When a user ran `fenn pull <bad-name> ...` for a template that does not exist, the CLI suggested `fenn pull --list`. That flag does not exist on the `pull` subcommand. The correct command for listing templates is `fenn list`.

## Changes

- `fenn/cli/pull.py`: point the template-not-found hint to `fenn list`.
- `tests/unit/cli/test_pull_command.py`: add regression assertions for the corrected hint.
- `fenn/experimental/vision/normalize.py`: replace 12 equivalent NumPy bound-method reductions (`array.max()` / `array.min()`) with `np.max(array)` / `np.min(array)`. These mechanical substitutions resolve the `ty` diagnostics that caused the PR pipeline to fail; no normalization behavior is intentionally changed.

## Scope

The PR is limited to 3 files with 17 additions and 13 deletions. Unrelated dashboard, template-test, normalization-validation, warning-handling, and lockfile changes have been removed.

## Verification

- `pre-commit run --all-files`: passed
- Targeted CLI and normalization tests: 53 passed
- Configured Nox unit session: 918 passed, 7 optional-dependency skips

## Reproduction

Run `fenn pull DOES_NOT_EXIST temp`.

Expected: the response suggests `fenn list`.

Previously observed: the response suggested the nonexistent `fenn pull --list` command.

### CODE REVIEW

**Comment by ApusBerliozi** [comment]
Hey @rusetiq! Thx 4 your work

Kindly check why pipeline fails

**Comment by rusetiq** [comment]
sure! give me a while, i'll work on it and commit

**Comment by rusetiq** [comment]
im pretty sure thatd fix it!
i changed five np calls in fenn/experimental/vision/normalize.py from instance methods like:
`alpha_channel.max()` to `np.max(alpha_channel)`
and similarly changed:
`rgb_channels_float.max()` to `np.max(rgb_channels_float)`

**Comment by rusetiq** [comment]
@ApusBerliozi fixed!

**Comment by ApusBerliozi** [comment]
Hey @rusetiq! Thx

I think, there are way to more changes in files for such an error. I will comeback with full report once I have some free time

**Comment by rusetiq** [comment]
Thanks for flagging the scope issue. I audited and reduced the PR from 12 files (+709/-144) to 3 files (+17/-13).

Removed from this PR:
- dashboard runner changes
- template-test redesign
- normalization behavior/edge-case changes
- image-batch and Ruff cleanup
- `uv.lock` regeneration

The branch now contains only the original CLI fix and regression assertions, plus the 12 mechanical `ndarray.min/max` to `np.min/max` substitutions required to clear the pipeline's `ty` diagnostics. Pre-commit passes, the targeted suite reports 53 passed, and the configured Nox unit session reports 918 passed with 7 optional-dependency skips.

Updated in commit `57a1b00`.

**Comment by rusetiq** [comment]
@ApusBerliozi mind checking?

File: fenn/experimental/vision/normalize.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def _normalize_minus1_1(array: np.ndarray) -> np.ndarray:
                 normalized_alpha = 2.0 * (alpha_channel / dtype_max) - 1.0
             else:
                 # Float type: if > 1, assume [0, 255] range, else already in [0, 1]
-                if alpha_channel.max() > 1.0:
+                if np.max(alpha_channel) > 1.0:
                     normalized_alpha = 2.0 * (alpha_channel / 255.0) - 1.0
```
Comment (ApusBerliozi) [inline fenn/experimental/vision/normalize.py:227]: Could you please elaborate, why we need to add changes here?

**Comment by ApusBerliozi** [comment]
Hey @rusetiq!

Sorry 4 taking so long, was busy. Kindly check review comment
